portfolio/src/main/webapp/style.css

```diff
@@ -1,22 +1,29 @@
-/* globals */
+/*************** globals ***************/
 
 html {
+
+    background-color: #f5f5f5;
+
+/* saving in case I want to use an image instead, will make a decision in the final product
```

Comment:
You can delete this since the code is saved in previous git commits. Commented out code hurts readability of code pretty quickly.